lib/spoom/cli/srb/sigs.rb

```diff
@@ -145,6 +145,7 @@ class Module; include T::Sig; end
           tapioca_context.write!("Gemfile", <<~RB)
             source "https://rubygems.org"
 
+            gem "rbs", ">= 4.0.0.dev.4"
```

Comment:
@KaanOzkan Can you add a comment in https://github.com/Shopify/spoom/blob/main/spoom.gemspec#L37 so we don't forget to update this line when we finally bump RBS? 🙏

Reply:
Good point https://github.com/Shopify/spoom/pull/814